prepare.backend.sh

```
@@ -0,0 +1,17 @@
+if [[ "${PERF_MODE:-}" ]]; then
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)